assets/css/styles.css

```diff
@@ -1,102 +1,135 @@
 }
 
 #planSaveIcon {
-    cursor: pointer;
+  cursor: pointer;
 }
 
 .svg-container {
-    display: flex;
-    justify-content: center;
+  display: flex;
+  justify-content: center;
 }
 
 .svg-content {
-    display: inline-block;
-    top: 0;
-    left: 0;
+  display: inline-block;
+  top: 0;
+  left: 0;
 }
 
 /*an option for a svg to be used directly*/
 .svg {
-    background-size: 100%;
-    height: 100vh;
-    width: 100%;
-    background: url("http://fofwebdesign.co.uk/template/_testing/test/svg/page-bg.svg") no-repeat 50px 50px;
-    color: transparent;
+  background-size: 100%;
+  height: 100vh;
+  width: 100%;
+  background: url("http://fofwebdesign.co.uk/template/_testing/test/svg/page-bg.svg")
+    no-repeat 50px 50px;
+  color: transparent;
 }
 
 /* 404 styles.css */
 .specific404 {
-        background-image: url("../../assets/public/aJTiW00qqtI-unsplash.jpg");
-        background-size: cover;
-        font-family: 'Roboto', 'sans-serif';
-        display: flex;
-        height: 100vh;
-        justify-content: center;
-        align-items: center;
-        flex-direction: column;
-    .header404{
-        font-family: 'Chango', cursive;
-        font-size: 48px;
-    }
-    .content404{
-        text-align: center;
-    }
-}
\ No newline at end of file
+  background-image: url("../../assets/public/aJTiW00qqtI-unsplash.jpg");
+  background-size: cover;
+  font-family: "Roboto", "sans-serif";
+  display: flex;
+  height: 100vh;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  .header404 {
+    font-family: "Chango", cursive;
+    font-size: 48px;
+  }
+  .content404 {
+    text-align: center;
+  }
+}
+
+.google-sign-in-button {
+  cursor: pointer;
+  transition: background-color 0.3s, box-shadow 0.3s;
+
+  padding: 12px 16px 12px 42px;
+  border: none;
+  border-radius: 3px;
+  box-shadow: 0 -1px 0 rgba(0, 0, 0, 0.04), 0 1px 1px rgba(0, 0, 0, 0.25);
+
+  color: #757575;
+  font-size: 14px;
+  font-weight: 500;
+  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen,
+    Ubuntu, Cantarell, "Fira Sans", "Droid Sans", "Helvetica Neue", sans-serif;
+
+  background-image: url(data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMTgiIGhlaWdodD0iMTgiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyI+PGcgZmlsbD0ibm9uZSIgZmlsbC1ydWxlPSJldmVub2RkIj48cGF0aCBkPSJNMTcuNiA5LjJsLS4xLTEuOEg5djMuNGg0LjhDMTMuNiAxMiAxMyAxMyAxMiAxMy42djIuMmgzYTguOCA4LjggMCAwIDAgMi42LTYuNnoiIGZpbGw9IiM0Mjg1RjQiIGZpbGwtcnVsZT0ibm9uemVybyIvPjxwYXRoIGQ9Ik05IDE4YzIuNCAwIDQuNS0uOCA2LTIuMmwtMy0yLjJhNS40IDUuNCAwIDAgMS04LTIuOUgxVjEzYTkgOSAwIDAgMCA4IDV6IiBmaWxsPSIjMzRBODUzIiBmaWxsLXJ1bGU9Im5vbnplcm8iLz48cGF0aCBkPSJNNCAxMC43YTUuNCA1LjQgMCAwIDEgMC0zLjRWNUgxYTkgOSAwIDAgMCAwIDhsMy0yLjN6IiBmaWxsPSIjRkJCQzA1IiBmaWxsLXJ1bGU9Im5vbnplcm8iLz48cGF0aCBkPSJNOSAzLjZjMS4zIDAgMi41LjQgMy40IDEuM0wxNSAyLjNBOSA5IDAgMCAwIDEgNWwzIDIuNGE1LjQgNS40IDAgMCAxIDUtMy43eiIgZmlsbD0iI0VBNDMzNSIgZmlsbC1ydWxlPSJub256ZXJvIi8+PHBhdGggZD0iTTAgMGgxOHYxOEgweiIvPjwvZz48L3N2Zz4=);
```

Comment:
is this the official url() ?

Reply:
Good question, we should probably use the [official one](https://developers.google.com/identity/branding-guidelines#render-html-button) .